### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by AbdusSattar-70**
### In this milestone, I have implemented the following:

- [ ]  Write plain HTML markup with minimum styling 
- [ ] Used webpack
- [ ] Used Gitflow
- [ ] Used wireframe as a guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,173 @@
...
+- **[Javascript]**
+- **[Webpack]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo<a name="live-demo"></a>
+
+- [Live Demo Link](https://abdussattar-70.github.io/Leaderboard/dist/)
```
Comment (Olamarx) [inline README.md]: - [ ] Since this project is not live yet, I would advise you to remove this live section.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @AbdusSattar-70, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has the live section in your readme without an actual live site.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md (hunk 1 above)
Comment (AbdusSattar-70) [reply README.md]: Hello  @Olamarx,
Thanks for your suggestion I will remove it.

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @AbdusSattar-70    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbdusSattar-70** [comment]
Hello @hiromataba,
Thanks for investing time in reviewing and approving.👌👌

**Pull request merged**